apps/examples/testcase/le_tc/network/tc_net_dup.c

```diff
@@ -1,130 +1,202 @@
 /****************************************************************************
- *
- * Copyright 2017 Samsung Electronics All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied. See the License for the specific
- * language governing permissions and limitations under the License.
- *
- ****************************************************************************/
-
-/// @file tc_net_dup.c
-/// @brief Test Case Example for dup() API
-#include <tinyara/config.h>
-#include <stdio.h>
-#include <errno.h>
-#include <sys/stat.h>
-#include <net/if.h>
-#include <arpa/inet.h>
-#include <netinet/in.h>
-#include <netutils/netlib.h>
-#include <sys/socket.h>
+*
```

Comment:
Your team modified this in almost pull request. Don't do that. Let's revert these kind of change.